README.md

```diff
@@ -460,22 +460,28 @@ directory or `make install` go install *aws-nuke* into `$GOPATH/bin`. With
 
 ### Docker
 
-To use the provided Docker implementation, first you need to make sure you've 
-created the config file and that it exists in the `config` directory.
-Make sure you are in the project's root directory, then you need to build 
-the image and run the container:
+You can run *aws-nuke* with Docker by using a command like this:
 
 ```bash
-$ docker build . -t rebuy-de/aws_nuke
-
-$ docker run -a stdin -a stdout -ti rebuy-de/aws_nuke -c config/nuke-config.yml --profile aws-nuke-example
+$ docker run \
+    --rm -it \
+    -v /full-path/to/nuke-config.yml:/home/aws-nuke/config.yml \
+    -v /home/user/.aws:/home/aws-nuke/.aws \
+    quay.io/rebuy/aws-nuke:v2.11.0 \
```

Comment:
```suggestion
    quay.io/rebuy/aws-nuke:latest \
```

Otherwise you will always have to update the README

Reply:
We only have `master` and I somehow would not like to suggest using it.

We would need to update the CI, if we want to support `latest`.

Reply:
A latest versions would be nice, not sure how hard it is to implement. We can't expect users to check the `Releases`  or quay.io page and change it accordingly.